integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datamap/TestDataMapCommand.scala

```diff
@@ -244,7 +244,7 @@ class TestDataMapCommand extends QueryTest with BeforeAndAfterAll {
   test("create pre-agg table with path") {
     sql("drop table if exists main_preagg")
     sql("drop table if exists main ")
-    val path = "./preAggTestPath"
+    val path = System.getProperty("java.io.tmpdir") + "/" + System.nanoTime + "_preAggTestPath"
```

Comment:
Can you point to target folder so that it will clean for every run.

Reply:
ok, done